Simulator/Edges/NG911/All911Edges.cpp

```diff
@@ -34,15 +34,15 @@ void All911Edges::setupEdges()
       W_.assign(maxTotalEdges, 0);
       type_.assign(maxTotalEdges, ETYPE_UNDEF);
       edgeCounts_.assign(numVertices, 0);
-      summationPoint_.assign(maxTotalEdges, nullptr);
+      summationPoint_.assign(maxTotalEdges, -1);
       destVertexIndex_.assign(maxTotalEdges, 0);
       sourceVertexIndex_.assign(maxTotalEdges, 0);
       inUse_ = make_unique<bool[]>(maxTotalEdges);
       fill_n(inUse_.get(), maxTotalEdges, false);
    }
 }
 
-void All911Edges::createEdge(const BGSIZE iEdg, int srcVertex, int destVertex, BGFLOAT *sumPoint,
+void All911Edges::createEdge(const BGSIZE iEdg, int srcVertex, int destVertex, const int sumPoint,
```

Comment:
Just out of curiosity, why `const int`? Is there a desire to emphasize that the method won't change this variable (which is passed by value)? While verbose, there might be some value here, but of course it raises the question of why we don't do it everywhere...

Reply:
My intention was to emphasize that the method won't change this variable. Also, few of the other APIs follow the same syntax, of marking an `int` as `const`. I agree with you that `int` is passed by value, and marking it as `const` is not a common practice. Maybe remove the `const` from the APIs?

Reply:
I'd say so. This of course makes one think that there are probably many circumstances where we pass variables by value to functions for which there is the mental model that this is input only and a constant, but we never mark them that way. I suppose the idea is that the function can't modify the caller's variable and so whether that local variable is treated as a constant or not isn't part of the interface, but rather implementation of that function. So, yes, let's remove unnecessary `const`s, unless there is some reason to specifically draw a person's attention to that (otherwise, it will, like it did here).